app/src/main/java/com/github/sdp/mediato/ui/viewmodel/UserViewModel.java

```diff
@@ -4,16 +4,20 @@
 import androidx.lifecycle.MutableLiveData;
 import androidx.lifecycle.ViewModel;
 
+import com.github.sdp.mediato.MainActivity;
 import com.github.sdp.mediato.data.UserDatabase;
 import com.github.sdp.mediato.model.Location;
 import com.github.sdp.mediato.model.User;
+import com.mifmif.common.regex.Main;
```

Comment:
I do not see the usage of this import in the new code. Is it still used ?

Reply:
mybad